daml-script/dump/src/main/scala/com/daml/script/dump/Config.scala

```diff
@@ -30,6 +43,18 @@ object Config {
       .required()
       .unbounded()
       .action((x, c) => c.copy(parties = x :: c.parties))
+    opt[String]("start")
+      .optional()
+      .action((x, c) => c.copy(start = parseLedgerOffset(x)))
+      .text(
+        "The transaction offset (exclusive) for the start position of the dump. Optional, by default the dump includes the beginning of the ledger."
+      )
+    opt[String]("end")
+      .optional()
+      .action((x, c) => c.copy(start = parseLedgerOffset(x)))
+      .text(
+        "The transaction offset (inclusive) for the end position of the dump. Optional, by default the dump includes the current end of the ledger."
```

Comment:
Do we have the invariant that `--start=x --end=x` always produces the acs dump at the given offset?